### PR TITLE
MongoDB Setup

### PR DESCRIPTION
**Pull request by ayalavarti**
Added MongoDB dependency to `pom.xml`. Created `User` class for testing purposes. Set up a 
`CommandController` class that allows for CRUD operations to an Atlas Cluster.

To run, start local instance of MongoDB* (using mongod), then navigate to /update?firstName=FIRST&lastName=LAST to add a user to the database and /users to show all users. Note: updated .jar file is now server-0.0.2.

*APIKeys are ignored, so create an `APIKeys class` in `uplink.db` that returns a MongoDB URI to the Atlas cluster you want to connect to.

Updated favicon to BSE logo!

### CODE REVIEW

File: server/src/main/java/models/Address.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package models;
+
+public class Address {
```
Comment (ali-mir) [inline server/src/main/java/models/Address.java]: Is this class for the address of the ground station? I would also add country (one of our ground stations is in Rome)

File: server/src/main/java/uplink/db/CommandController.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+import models.User;
+
+@RestController
+public class CommandController {
+
+	MongoClientURI uri = new MongoClientURI(APIKeys.getMongoDBPassword());
```
Comment (ali-mir) [inline server/src/main/java/uplink/db/CommandController.java:22]: Do you mind sending over the code for the APIKeys class? Just so everyone can have the same structure.

**Review by ali-mir — CHANGES_REQUESTED** [review]
Great job! Just a few comments. Feel free to address them and then request another review!

File: server/src/main/java/uplink/db/CommandController.java (hunk 2 above)
Comment (ayalavarti) [reply server/src/main/java/uplink/db/CommandController.java:22]: Just sent it to the Slack channel.

File: server/src/main/java/models/Address.java (hunk 1 above)
Comment (ayalavarti) [reply server/src/main/java/models/Address.java]: Sorry about that! I made that class just to test out creating a document to upload to MongoDB. I'll delete that from the models package.

**Comment by ayalavarti** [comment]
@ali-mir Just pushed a new commit to this branch to remove the unnecessary Java classes.

**Review by ali-mir — APPROVED** [review]
LGTM! Good job!

**Pull request merged**